ts/utils/paymentMethodCapabilities.ts

```diff
@@ -25,33 +31,49 @@ export const canMethodPay = (paymentMethod: PaymentMethod): boolean => {
 };
 
 /**
- * check if a payment method is supported or not
+ * Return a custom representation for a payment method who cannot pay
+ * @param paymentMethod
+ */
+const paymentNotSupportedCustomRepresentation = (
+  paymentMethod: PaymentMethod
+): PaymentSupportStatus => {
+  switch (paymentMethod.kind) {
+    case "CreditCard":
+      return paymentMethod.info?.issuerAbiCode &&
+        paymentMethod.info.type !== TypeEnum.PRV
```

Comment:
```suggestion
      // if it is a cobadge
      return paymentMethod.info?.issuerAbiCode &&
        paymentMethod.info.type !== TypeEnum.PRV
```
if I well understood

Reply:
Changed in ee549cabecef18957f5cc4ec382c3a0146ba3cc4 in order to have a more explicit function that explains.